libraries/joomla/form/fields/civiperms.php

```diff
@@ -49,9 +52,18 @@ protected function bootstrapCivi() {
    * the issue ID in this comment and reference it in the code as well.
    */
   protected function getInput() {
-    JHtml::_('bootstrap.tooltip');
+    HTMLHelper::_('bootstrap.tooltip');
```

Comment:
Change looks good but breaks J3 compatibility? Move into `if` structure below if we want to maintain this for the time being?